tests/Integration/Behaviour/Features/Scenario/Attribute/list_attribute_groups.feature

```diff
@@ -0,0 +1,67 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s attribute --tags list-attribute-group
+@reset-database-before-feature
+@list-attribute-group
+Feature: Attribute Group
+  PrestaShop allows BO users to list attribute groups
+  As a BO user
+  I should be able to list attribute groups
+
+  Background:
+    Given language with iso code "en" is the default one
+    And attribute group "size" named "Size" in en language exists
+    And attribute group "color" named "Color" in en language exists
+    And attribute group "dimension" named "Dimension" in en language exists
+    And attribute group "paper_type" named "Paper Type" in en language exists
+    And attribute "s" named "S" in en language exists
+    And attribute "m" named "M" in en language exists
+    And attribute "l" named "L" in en language exists
+    And attribute "xl" named "XL" in en language exists
+    And attribute "grey" named "Grey" in en language exists
+    And attribute "taupe" named "Taupe" in en language exists
+    And attribute "beige" named "Beige" in en language exists
+    And attribute "white" named "White" in en language exists
+    And attribute "off_white" named "Off White" in en language exists
+    And attribute "red" named "Red" in en language exists
+    And attribute "black" named "Black" in en language exists
+    And attribute "camel" named "Camel" in en language exists
+    And attribute "orange" named "Orange" in en language exists
+    And attribute "blue" named "Blue" in en language exists
+    And attribute "green" named "Green" in en language exists
+    And attribute "yellow" named "Yellow" in en language exists
+    And attribute "brown" named "Brown" in en language exists
+    And attribute "pink" named "Pink" in en language exists
+
+  # NOTE: we don't initialize specific data since we already have the core features
+  Scenario: List attribute groups
+    When I list all attribute groups I should get following results:
```

Comment:
the action formulation is wrong. (it is mixed between action and the outcome).
It should be like this
```
Given|When|Then I should see a following list of attributes
```